app/components/works/edit/attached_file_component.html.erb

```diff
@@ -0,0 +1,22 @@
+<tr class="attached-files <%= 'dz-complete' if uploaded? %>" data-dropzone-target="preview">
+    <%= form.fields_for :attached_files, attached_file do |file_form| %>
+    <td class="file-level" style="--depth: <%= depth+1 %>rem" data-dropzone-target="fileName">
+        <span class="fa-regular fa-file" aria-hidden="true"></span> <%= basename %>
+    </td>
+    <td>
+        <%= file_form.text_field :label, class: 'form-control' %>
+    </td>
+    <td>
+        <%= file_form.check_box :hide, style: 'align: center' %>
+    </td>
+    <td>
+        <%= button_tag type: 'button',
+                    class: "dz-remove pull-right btn remove-file",
+                    "aria-label": "Remove file",
+                    data: { action: "click->dropzone#removeAssociation" } do %>
+        <span class="fa-regular fa-trash-alt"></span>
+        <% end %>
+        <%= file_form.hidden_field :_destroy %>
+    <% end %>
```

Comment:
I expected this `end` statement to be just inside the `</tr>` rather than the `</td>`. Typo? Intentional?

Reply:
Oddly enough, this was the fix to get the form rendering correctly within the table, so it's intentional and required.  I will add a comment about this.

I found a hint in this stack overflow question: https://stackoverflow.com/questions/6602579/fields-for-in-table-produces-technically-incorrect-html

Reply:
Huh!